src/app/views/query-runner/request/Request.tsx

```diff
@@ -19,13 +21,51 @@ import { translateMessage } from '../../../utils/translate-messages';
 import { convertVhToPx } from '../../common/dimensions-adjustment';
 import { Auth } from './auth';
 import { RequestBody } from './body';
+import { loadAndInitialize } from './feedback/feedbackWrapper';
 import { RequestHeaders } from './headers';
 import { Permission } from './permissions';
 import './request.scss';
 
 export class Request extends Component<IRequestComponent, any> {
   constructor(props: IRequestComponent) {
-    super(props);
+    super(props)
+    this.state = {
+      officeBrowserFeedback: undefined,
+      enableShowSurvey: false,
+    }
+    // this.setOfficeBrowserFeedbackUtility = this.setOfficeBrowserFeedbackUtility.bind(this);
+    this.showCustomSurvey = this.showCustomSurvey.bind(this);
```

Comment:
You can avoid binding methods in the constructor if you define the method as a variable  in the class.

```
showCustomSurvey = () => {}
```